client/src/components/Doctor/DoctorDashboard.tsx

```diff
@@ -1,35 +1,164 @@
-import React, { FunctionComponent, useContext } from "react";
+import axios from "axios";
+import React, { ChangeEvent, FunctionComponent, useContext, useEffect, useState } from "react";
 import { useNavigate } from "react-router-dom";
+import Web3 from "web3";
+import { ethers } from "ethers";
 import { AuthContext } from "../../providers/AuthProvider";
+import { HealthContext } from "../../providers/HealthProvider";
+import { healthBlockABI, healthBlockAddress } from "../../config/constants";
+import Web3Modal from 'web3modal';
+import { HealthBlock__factory } from '../../contracts/factories/HealthBlock__factory';
+//import crypto from 'crypto';
+
+const DoctorDashboard: React.FunctionComponent<{}> = () => {
 
-const DoctorDashboard: FunctionComponent<{}> = () => {
-    const navigate = useNavigate();
     const { user, role, logout } = useContext(AuthContext);
+    const { handleRaiseRequest, handleVerifyDoctor} = useContext(HealthContext);
+    const [specialization, setspecialization] = useState<string>("");
+    const [medicalInstitution, setMedicalInstitution] = useState<string>("");
+    
+    const navigate = useNavigate();
+    //const [contract, setContract] = useState<any>();
+    const [file, setFile] = useState<File | null>(null);
+
     const logouthandler = () => {
         logout?.();
         navigate('/doctorlogin');
     }
+
+    const handleFileUpload = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
encrypt files being uploaded to s3 as doctor credentials are confidential

Reply:
It will be addressed as part of HEAL 83 & 84